### ISSUE TITLE
Can't get claimed accuracy (81.1%) with the provided configs? 

### ISSUE THREAD

**Issue by abdurad** [GenjiB/LAVISH#9]
After fixing few bugs I managed to reach ~79% accuracy only, I used the provided settings:
python3 main_trans.py --Adapter_downsample=8 --audio_folder=$PATH/raw_audio --batch_size=2 --early_stop=5 --epochs=50 --is_audio_adapter_p1=1 --is_audio_adapter_p2=1 --is_audio_adapter_p3=0 --is_before_layernorm=1 --is_bn=1 --is_fusion_before=1 --is_gate=1 --is_post_layernorm=1 --is_vit_ln=0 --lr=5e-05 --lr_mlp=4e-06 --mode=train --num_conv_group=2 --num_tokens=2 --num_workers=16 --video_folder=$PATH/video_frames  --is_multimodal=1 --vis_encoder_type=swin

The code works on the validation set and the maximum accuracy is 78%, while the accuracy for the test set is 79%. Could you please provide the settings for reported results?

**Comment by GenjiB** [comment]
The gap is huge. Did you use the process images and audio I provided?

If, so. Let me check if any bug in it. and will keep you updated in a few days.

Thank you

**Comment by abdurad** [comment]
Yes, I have used the prcessed images and audio provided in the huggingface hub.

**Comment by GenjiB** [comment]
@abdurad got it. I can get the same results on the machine where I used in the paper. I am trying to make reproduceability  more reliable on different machines. I will update the codes as soon as possible.

**Comment by GenjiB** [comment]
@abdurad  I updated the code. Let me know if you can get the similar results
